### PR TITLE
Fixed plotting

### PR DESCRIPTION
**Pull request by MonsterMarkus**
Fixed the live plots

### CODE REVIEW

File: Monster.m [hunk 1]
```diff
@@ -132,7 +132,10 @@
 
 			obj.Logger.log('(MONSTER - run) downlink UE data decoding', 'DBG');
 			obj.downlinkUeDataDecoding();
-
+            
+            obj.Logger.log('(MONSTER - run) plotting constalation diagrams and Spectrum', 'DBG');
```
Comment (jakthra) [inline Monster.m]: Typo here

File: setup/MonsterConfig.m [hunk 2]
```diff
@@ -207,13 +207,13 @@
...
 			% Properties related to plotting
 			Plot = struct();
-			if obj.SimulationPlot.runtimePlot
-				Plot.Layout = '';
-				Plot.LayoutFigure = '';
-				Plot.LayoutAxes = axes;
-				Plot.PHYFigure = '';
-				Plot.PHYAxes = axes;
-			end
+			%if obj.SimulationPlot.runtimePlot
```
Comment (jakthra) [inline setup/MonsterConfig.m]: Are these not needed anymore?

Comment (MonsterMarkus) [reply setup/MonsterConfig.m]: They made an empty figure, where nothing happened.

File: Monster.m (hunk 1 above)
Comment (MonsterMarkus) [reply Monster.m]: Fixed

**Pull request merged**